src/components/PostDetails.tsx

```diff
@@ -1,106 +1,110 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Post } from '../types/Post';
+import { DispatchContext, StateContext } from '../Store';
+import { deleteComment, getComments } from '../api/comments';
+
+type Props = {
+  selectedPost: Post;
+};
+
+export const PostDetails: React.FC<Props> = ({ selectedPost }) => {
+  const { id, title, body } = selectedPost;
+  const [load, setLoad] = useState(false);
```

Comment:
Rename all boolean variables using this [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isLoading, setIsLoading] = useState(false);
```